src/components/subscriptions/licenses/LicenseManagementModals/LicenseManagementRevokeModal.jsx

```diff
@@ -90,11 +103,26 @@ const LicenseManagementRevokeModal = ({
     }
     setRequestState({ ...initialRequestState, loading: true });
     const makeRequest = () => {
-      if (revokeAllUsers) {
+      const filtersPresent = checkIfFiltersPresent(filtersAndValues)
```

Comment:
filters or not, if a user had clicked "select all", whenever they unclick a specific user, isEntireTableSelected gets set to false (which flows though to this file as `revokeAllUsers`), so the "unclick" scenario will never hit revoke-all endpoint or the bulk-revoke (with filters), and will only hit bulk-revoke with user_emails

The addition of filtersPresent will protect us again the "has filter, selects all, revokes every single license". Once the backend implements filters as an option, then in the case where the "entire table is selected" with a filter, we simply pass that filter to the backend and perform the action on (hopefully) the same licenses. Deeefinitely going to need to have some thorough testing here